charts/consul/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: consul
-version: 0.43.0
+version: 0.43.1
```

Comment:
I think it should be 0.44 because there aren't just bug fixes in the changelog

Reply:
ooh!  thank you!